app/controllers/ticket_purchases_controller.rb

```diff
@@ -4,8 +4,15 @@ class TicketPurchasesController < ApplicationController
   authorize_resource :conference_registrations, class: Registration
 
   def create
+    if(params[:tickets].nil?)
```

Comment:
This is the case when ticket purchase is already created but unpaid. When the user click the pay button on ticket purchase index page, this method is called but there will be no params in this case as the ticket purchase is already created and we just need to redirect the user to new payment controller method.